src/main/scala/io/iohk/ethereum/jsonrpc/EthJsonMethodsImplicits.scala

```diff
@@ -137,6 +137,21 @@ object EthJsonMethodsImplicits extends JsonMethodsImplicits {
         t.transactionResponse.map(Extraction.decompose).getOrElse(JNull)
     }
 
+  implicit val eth_d  =
```

Comment:
Minor detail: Maybe we could name this `val` as `eth_getTransactionByBlockNumberAndIndex`?